server/db/models/book.js

```diff
@@ -13,10 +13,6 @@ export default (sequelize, DataTypes) => {
       allowNull: false,
       type: DataTypes.NUMERIC
     },
-    authorID: {
```

Comment:
WHy was this removed?

Reply:
I removed it there, to re-defined it in the book-migration file line 23. I read from the link below that if you have a foreignkey defined in both the model and the migration files. sequelize will throw an error. So it also has to be defined in the migration file of that model. 
https://www.duringthedrive.com/2017/05/06/models-migrations-sequelize-node/

Reply:
I see